POEM_072.md

```diff
@@ -0,0 +1,72 @@
+POEM ID: 072  
+Title: Add ability to modify bounds and scaling of implicit outputs and optimizer variables after creation.  
+authors: @robfalck  
+Competing POEMs:  
+Related POEMs:  
+Associated implementation PR: N/A  
+
+
+Status:
+
+- [x] Active
+- [ ] Requesting decision
+- [ ] Accepted
+- [ ] Rejected
+- [ ] Integrated
+
+
+## Motivation
+
+In OpenMDAO, implicit outputs can have bounds attach to them that are enforced by bounds-enforcing linesearches within nonlinear solvers.
```

Comment:
Any output can have bounds. Not just implicit

Reply:
True, will correct